app/gui/language/ast/macros/src/lib.rs

```diff
@@ -110,7 +109,7 @@ fn mk_product_type(
     let ident_nested = format!("{}{}", decl.ident, variant.ident);
     let ident_nested = Ident::new(&ident_nested, Span::call_site());
     let ident = if is_flat { ident_flat } else { ident_nested };
-    let generics = syn::Generics { params, ..default() };
+    let generics = syn::Generics { params, ..Default::default() };
```

Comment:
why this change? How does it affect the time?

Reply:
This is a consequence of removing prelude import. I could also import `default`, but it would require enabling extra nightly feature just for one usage.

Reply:
Ok, why the prelude was removed here, to make it has less dependencies?

Reply:
It's a proc-macro crate. Having prelude imported in any of them would result in a lot of crates being compiled twice (once for native target, once for wasm). If we want macro prelude, we should make a separate crate for it. But I don't think it's necessary right now.